portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +26,70 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function expandDesign(clickedId){
+//whenever you click an image on the description page, this pulls up more images in the category alongside descriptions
+const designPage = document.getElementById('design_container');
+designPage.innerHTML = "";
+console.log(clickedId); //REMOVE WHEN DONE
+
+//descriptions for every single page that can be clicked on
+const descriptions = 
+    ["Parasite is a fake band created by me and my friends back home."  
+    +"While we have not created any music yet (hence the fake band), "
+    +"I have created plenty of promotional material for us. "
+    +"Hopefully one day the band can become a reality.",
+    "I primarily work in traditional art, as seen by this selection of projects."
+    +"Most of this art was created for my AP Art portfolio, but some pieces"
+    +"have been created since then. Hopefully over this break I can do more"
+    +"traditional art."
+    ]
+
+if(clickedId == "fake_band"){
+    //initial description
+    var title = document.createElement("h1");
+    var titleText = document.createTextNode("Parasite");
+    title.appendChild(titleText);
+    var desc = document.createElement("p");
+    desc.className = "main_desc";
+    var descText = document.createTextNode(descriptions[0]);
+    desc.appendChild(descText);
+    designPage.appendChild(title);
+    designPage.appendChild(descText);
+
+    //creating the picture and description layout
+    var outer_div = document.createElement("div");
+    outer_div.className = "column_disp";
+    const img_descriptions = 
+        [
+            "The first image is of a record I designed while playing with shape language."
+            + "While simple, it utilizes our primary color scheme of red, yellow, and green"
+            + "and was really fun to make. Also, the drop shadow makes it really pop. If we"
+            + "were a real band, this would probably be the design for our vinyls.",
+            "placeholder",
+            "placeholder"
+        ];
+    const img_ids = 
```

Comment:
I'm a bit unsure if these ids. based on their usage, we're creating divs and giving them an id based on these strings. However, if we don't need to reference them by ID, then we don't need to set it. So that makes the IDs most useful as something to iterate over. However, we could just use the list of descriptions as a way to make sure we iterate the correct number of times.

Reply:
I have these IDs so I can set the background of the div to the image. This way you cannot drag any of the images off of the page to save them and everything can be sized easily. I redid the way these were organized in the latest version, also.